tasks/jscs.js

```diff
@@ -0,0 +1,18 @@
+module.exports = {
+  js: {
+    options: {
+      config: '.jscsrc',
+      esnext: true,
+      // fix: false,
```

Comment:
Oh, yes, sorry. It's a neat feature that automatically fixes easy-to-fix nits. But I don't think it's needed and it's a bit error-prone (I tested it); I'll take it out.